boilerplate/flyte/golang_support_tools/go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/flyteorg/boilerplate
 
-go 1.17
+go 1.16
```

Comment:
Why are we downgrading here

Reply:
it's boilerplate changes, Need to fix in boilerplate

Reply:
Ok what should be the plan to fix this.Does this require another round of PR for all repos

Reply:
boilerplate will create pr, I will create a pr for that